doc/user/content/sql/create-connection.md

```diff
@@ -8,19 +8,123 @@ menu:
 
 ---
 
-A connection describes how to connect and authenticate to an external system you want Materialize to read data from. Once created, a connection is **reusable** across multiple [`CREATE SOURCE`](/sql/create-source) and [`CREATE SINK`](/sql/create-sink)
-statements.
+A connection describes how to connect and authenticate to an external system you want Materialize to read data from. Once created, a connection is **reusable** across multiple [`CREATE SOURCE`](/sql/create-source), [`CREATE SINK`](/sql/create-sink), and `CREATE CONNECTION` statements.
```

Comment:
I assume you added `CREATE CONNECTION` to cover CSR, SSH and PrivateLink connections, but this makes it pretty confusing. It should be enough to mention it under each of these connection types.